be/src/storage/rowset/block_split_bloom_filter.h

```diff
@@ -45,6 +45,18 @@ namespace starrocks {
 // Bloom filter is 32 bytes to take advantage of 32-byte SIMD instruction.
 class BlockSplitBloomFilter : public BloomFilter {
 public:
+    enum mode {
+        DEFAULT = 0,
+        PARQUET = 1,
```

Comment:
Is it better to have a `ParquetBlockSplitBloomFilter` derived from `BlockSplitBloomFilter` instead?